src/main/java/org/broadinstitute/hellbender/tools/exome/germlinehmm/CopyNumberTriStateHiddenMarkovModel.java

```diff
@@ -115,72 +113,62 @@ public double logTransitionProbability(final CopyNumberTriState currentState, fi
         Utils.nonNull(fromTarget);
         Utils.nonNull(nextPosition);
         final double distance = calculateDistance(fromTarget, nextPosition);
-        return logTransitionProbabilityCache.logProbability((int) distance, nextState, currentState);
+        double logProb = Double.NEGATIVE_INFINITY;
+        try {
+            logProb = logTransitionProbabilityCache.logProbability((int) distance, nextState, currentState);
+        } catch (NullPointerException ex) {
```

Comment:
Oh this was my half-assed way of debugging things, I forgot to remove it before PR ;-)